app/src/main/java/org/systers/mentorship/models/Task.kt

```diff
@@ -0,0 +1,34 @@
+package org.systers.mentorship.models
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * This data class represents a task related to a mentorship
+ * relation.
+ *
+ * @param id The id of this task
+ * @param description The description of this task
+ * @param isDone Represents whether this task has been
```

Comment:
`completed` word is in the line below, unnecessarily, right?
Can you fix that in the lines below

Reply:
Yep, fixed now.